query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/fields/config/LdapConfigurationField.java

```diff
@@ -50,17 +53,27 @@
   public LdapConfigurationField() {
     super(DEFAULT_FIELD_NAME, FIELD_TYPE_NAME, DESCRIPTION);
     pid = new PidField();
-    connection = new LdapConnectionField();
+    connection = new LdapConnectionField.ListImpl();
+    loadBalancing = new LdapLoadBalancingField();
     bindUserInfo = new LdapBindUserInfo();
     settings = new LdapDirectorySettingsField();
     claimMappings = new ClaimsMapEntry.ListImpl();
   }
 
   // Field getters
-  public LdapConnectionField connectionField() {
+  public ListField<LdapConnectionField> connectionField() {
```

Comment:
✏️ This and the following methods should be updated to be plural now that it returns a list.